spec/features/vote/javascripts/user_can_upvote_downvote_ajax_spec.rb

```diff
@@ -0,0 +1,145 @@
+require "rails_helper"
+
+feature 'user votes', %{
+
+  As an authenticated user
+  I want to upvote or downvote reviews
+  So that reviews can be ranked without reloading page
+
+  Acceptance Criteria
+  [x] I can upvote a review without reloading page
+  [x] I cannot upvote if I am not signed in
+
+
+  Acceptance Criteria
+  [x] I can downvote a review without reloading page
+  [x] I cannot downvote if I am not signed in
+
+
+  Acceptance Criteria
+  [x] I can cancel an upvote on a review without reloading page
+
+
+  Acceptance Criteria
+  [x] I can cancel an downvote on a review without reloading page
```

Comment:
You should just lump all these acceptance criteria together into one big list! No need to have the `Acceptance Criteria` heading four times